toolkit/build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
```

Comment:
Maybe a disclaimer that this is intended for local dev use, no warranty, subject to change etc. etc.

Reply:
sure ..